__tests__/app.test.js

```diff
@@ -72,6 +72,38 @@ describe("/api/categories", () => {
   });
 });
 
+describe("/api/reviews", () => {
+  test("GET - status: 200 - returns a reviews array of review object, each of which should have the corresponding properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        body.reviews.forEach((review) => {
```

Comment:
When using a forEach, we should also be asserting an expected length. How many reviews are in the database, and are we getting them all?